pandas/tests/indexes/test_datetimelike.py

```diff
@@ -1,467 +0,0 @@
-# -*- coding: utf-8 -*-
-
-import numpy as np
```

Comment:
and removed all of this (again just fine). that's why I thought you were moving Period. But can certainly finish in another.